src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ComboBox.cs

```diff
@@ -718,4 +794,378 @@ protected override void OnCollectionChanged(NotifyCollectionChangedEventArgs e)
 		}
 
 	}
+#if __IOS__
+	#region "Helper classes for the Modal Pop Up"
```

Comment:
they are just pages like any others; popup is not the best name, and it has special meaning in uwp...
```suggestion
	#region "Helper classes for the iOS Modal Page (UIModalPresentationStyle.pageSheet)"
```